_includes/sections/mobile-operating-systems.html

```diff
@@ -16,9 +16,9 @@ <h1 id="mobile_os" class="anchor"><a href="#mobile_os"><i class="fas fa-link anc
 %}
 
 {% include cardv2.html
-  title="LineageOS"
+  title="LineageOS (Official Builds)"
   image="/assets/img/svg/3rd-party/lineageos.svg"
-  description='LineageOS is a free and open-source operating system for smartphones and tablets, based on the official releases of the Android Open Source Project. It is the continuation of the CyanogenMod project.'
+  description='LineageOS is the a free and open-source successor to the CyanogenMod project. They have <a href="https://wiki.lineageos.org/devices/">official support for a variety of devices</a>, based on clean releases of the Android Open Source Project. For security reasons, we do not recommend third-party builds of LineageOS for unsupported devices.'
```

Comment:
```suggestion
  description='LineageOS is a free and open-source successor to the CyanogenMod project. They have <a href="https://wiki.lineageos.org/devices/">official support for a variety of devices</a>, based on clean releases of the Android Open Source Project. For security reasons, we do not recommend third-party builds of LineageOS for unsupported devices.'
```

Reply:
or... 
```suggestion
  description='LineageOS is the free and open-source successor to the CyanogenMod project. They have <a href="https://wiki.lineageos.org/devices/">official support for a variety of devices</a>, based on clean releases of the Android Open Source Project. For security reasons, we do not recommend third-party builds of LineageOS for unsupported devices.'
```

which is more accurate?

Reply:
And good point, “the” makes more sense.